library/packages/src/modules/Package.rb

```diff
@@ -103,6 +106,32 @@ def main
       @removed_packages = []
     end
 
+    # Determines whether the package is provided or not
+    #
+    # This method checks whether any installed package provides the given "package".
+    #
+    # @param package [String] Package name
+    # @param target [Symbol,nil] :autoinst or :system. If it is nil,
```

Comment:
I'm a bit confused by the possible values here. The description of the class it says this can work with either `PackageSystem` (libzypp) or `PackagesProposal`. So I was kind of expecting the values :system and :proposal (or something similar). But I see :autoinst instead that results in calls to `PackagesAI` (that is not exactly `PackagesProposal`).